smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/ContentLengthMiddleware.kt

```diff
@@ -20,17 +20,20 @@ class ContentLengthMiddleware(val model: Model, private val alwaysIntercept: Boo
     override fun render(
         writer: SwiftWriter,
         op: OperationShape,
-        operationStackName: String
+        operationStackName: String,
     ) {
         val hasHttpBody = MiddlewareShapeUtils.hasHttpBody(model, op)
         if (hasHttpBody || alwaysIntercept) {
-            writer.write(
-                "\$L.\$L.intercept(position: \$L, middleware: \$N())",
-                operationStackName,
-                middlewareStep.stringValue(),
-                position.stringValue(),
-                ClientRuntimeTypes.Middleware.ContentLengthMiddleware
-            )
+            val middlewareArgs = if (requiresLength || unsignedPayload) {
+                "requiresLength: $requiresLength, unsignedPayload: $unsignedPayload"
+            } else {
+                ""
+            }
```

Comment:
I think this could be written in a more idiomatic way by doing:
```
val str = "requiresLength: $requiresLength, unsignedPayload: $unsignedPayload"
val middlewareArgs = str.takeIf {requiresLength || unsignedPayload} ?: ""
```
*Factored out `str` to separate val just to avoid line breaks in one long statement.